src/components/EmptyArea.tsx

```diff
@@ -1,11 +1,10 @@
 import { Box, Center, VStack, Text, HTMLChakraProps } from "@chakra-ui/react"
-import { ReactFragment } from "react"
 
 import { EmptyBoxImage } from "assets"
 
-export interface EmptyAreaOption extends HTMLChakraProps<"div"> {
+export interface EmptyAreaProps extends HTMLChakraProps<"div"> {
   isItemEmpty: boolean
```

Comment:
could we make this prop optional?

Reply:
to be discussed under here [instead](https://github.com/isomerpages/isomercms-frontend/pull/993#discussion_r959316420)

Reply:
(repost because Idk why the github is duplicating shit ._.)
hmm I disagree here: isItemEmpty is used [here](https://github.com/isomerpages/isomercms-frontend/blob/add-empty-state-for-empty-resource/src/components/EmptyArea.tsx#L22) If I were to make this optional, then developers might not pass this parameter when calling EmptyArea, which then renders this whole component useless.